nautobot/extras/migrations/0117_task_queue_to_job_queue_migration.py

```diff
@@ -11,10 +12,13 @@ def migrate_task_queues_to_job_queues(apps, schema):
     ScheduledJob = apps.get_model("extras", "ScheduledJob")
     JobQueue = apps.get_model("extras", "JobQueue")
 
+    # should always create default job queue
+    JobQueue.objects.get_or_create(name="default", defaults={"queue_type": JobQueueTypeChoices.TYPE_CELERY})
     for job in Job.objects.all():
-        task_queues = job.task_queues
+        task_queues = job.task_queues or [settings.CELERY_TASK_DEFAULT_QUEUE]
```

Comment:
For this case here, if we do not have any `task_queues` assigned to the job class meta and we use `[settings.CELERY_TASK_DEFAULT_QUEUE]` for the job model should `job_queues_override` to be set to True at this point?

Reply:
I don't think it's needed, no, an undefined `task_queues` implies `["default"]`, it's not really an override.

Reply:
My issue is if the user goes to the edit page and does not check the `job_queues_override` checkbox or just does not do anything. The default queue will be cleared out which is not ideal imo.

Reply:
I *think* I fixed this in fccbd18 but please do double-check me.

Reply:
Yep it's fixed thank you!